Makefile

```diff
@@ -1,36 +1,36 @@
-.DEFAULT_GOAL: build
+.DEFAULT_GOAL: kyverno
 
 ##################################
 # DEFAULTS
 ##################################
+PWD := $(CURDIR)
 GIT_VERSION := $(shell git describe --match "v[0-9]*")
 GIT_BRANCH := $(shell git branch | grep \* | cut -d ' ' -f2)
 GIT_HASH := $(GIT_BRANCH)/$(shell git log -1 --pretty=format:"%H")
 TIMESTAMP := $(shell date '+%Y-%m-%d_%I:%M:%S%p')
-CONTROLLER_GEN=controller-gen
-CONTROLLER_GEN_REQ_VERSION := v0.4.0
+# VERSION is used to be able to set the version during a release through an env variable.
 VERSION ?= $(shell git describe --match "v[0-9]*")
```

Comment:
Duplicate with `GIT_VERSION := $(shell git describe --match "v[0-9]*")`.

Reply:
Interesting. Is this used in GitHub actions or where do we use it?

Reply:
Yes it's used in `release.yaml` in the `create-release` step.